src/lint.ts

```diff
@@ -44,6 +44,13 @@ export interface LintResult {
     output: string;
 }
 
+export interface ILinterOptionsRaw {
```

Comment:
Makes sense to me, let's keep this then. It shouldn't cause any backwards compat issues for library users as fulfilling the previous `ILinterOptions` interface will still fulfill the `ILinterOptionsRaw` interface. Maybe we should make the `formattersDirectory` field optional on both though, since it really is optional